src/browser/location.ts

```diff
@@ -82,6 +95,35 @@ export const doUpdateLocation = command(
     )
 );
 
+/**
+ * A query that never fails and returns `true` if there's a pending (blocked) location update
+ */
+export const pendingUpdateLocation = query(() =>
+  taskEither.of<void, boolean>(!!_historyBlockCallback)
+)(
+  refetch<void, void, boolean>(
+    setoidStrict as any,
```

Comment:
why these any?

Reply:
good point, removed as it is not needed in this case